maths/Quadratic Equations(Complexes Numbers)

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
@harshildarji This repo no longer supports legacy Python so we should avoid adding those `__future__`  imports which are no longer relevant in Python 3.  Adding them give the impression that we tested these scripts on legacy Python which we do not do.

Reply:
@cclauss Looking at the failure logs, I thought this could solve the issue. My bad! I'll remove it and see if it still gives the error!
Thanks!

Reply:
The problem may be that the filename contains spaces and uppercase characters so validate_filenames will fail.

Reply:
You're right! I'll look into this!

Reply:
@cclauss  Please review #1102. The main problem indeed was the filename as it was without proper `.py` extension. I have changed the filename as well as removed the `from __future__ import print_function` line from the code.